lib/tasks/test_self_service.rake

```diff
@@ -1,6 +1,5 @@
 namespace :test do
   namespace :self_service do
-    desc "Setup environment for self_service tests"
```

Comment:
This just removes the docstring from 'rake -T'; it's the same with the brakeman "setup" task (task is present, but no description so it's hidden in output)

Sent from my iPhone

> On Dec 8, 2015, at 3:28 PM, Jason Frey <notifications@github.com> wrote:
> 
> In lib/tasks/test_self_service.rake:
> 
> > @@ -1,6 +1,5 @@
> >  namespace :test do
> >    namespace :self_service do
> > -    desc "Setup environment for self_service tests"
> Why was this removed?
> 
> —
> Reply to this email directly or view it on GitHub.
>